_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -85,8 +86,20 @@ class JetpackTerminationDialogFeatures extends Component {
 		);
 	}
 
+	renderConnectedPlugins( plugins ) {
+		return (
+			<ul>
+				{ plugins
+					.filter( el => 'jetpack' !== el.slug )
+					.map( el => (
+						<li key={ el.slug }>{ el.name }</li>
```

Comment:
Should we use `plugin` instead of `el` to be as explicit as possible and make the file a bit more readable?